doc/user: Refresh the Getting Started guide (#18859)
MaterializeInc/materialize: doc/user/content/free-trial-faqs.md

---
title: "Free trial FAQs"
description: "Answers to frequency asked questions about Materialize free trials"
menu:
  main:
    parent: "quickstarts"
    weight: 15
---

> When you [sign up for Materialize](https://materialize.com/register/), you get a free trial account so you can explore the product without worrying about initial costs. This page answers some frequently asked questions about free trials.

Reviewer: ```suggestion
When you [sign up for Materialize](https://materialize.com/register/), you get a free trial account so you can explore the product and start building! This page answers some frequently asked questions about free trials.
```

Reviewer: > so you can explore the product without worrying about initial costs.

This makes me iffy. It should be focused on the value, rather than the cost.

Author: Changed.

## What's included in a free trial?

In Materialize, [cluster replicas](/overview/key-concepts/#cluster-replicas) are the physical resources for doing computational work. Cluster replicas come in various sizes, and each size corresponds to a specific number of [compute credits](https://materialize.com/pricing/).

> During your free trial, you get up to 4 compute credits per region at any given time. This limit should accommodate most trial scenarios. If you need additional resources during your trial, please contact sales@materialize.com.

Reviewer 2: ```suggestion
During your free trial, your credit consumption rate in a given region cannot exceed 4 credits per hour at any given time. This limit should accommodate most trial scenarios. If you need additional resources during your trial, please contact sales@materialize.com.
```

Author: Done. This is much more precise, but I do worry that it's nonetheless complex to digest. think I might add an example.

> During your free trial, your rate of credit consumption in a given region cannot exceed 4 credits per hour at any point in time. This limit should accommodate most trial scenarios. If you need additional resources during your trial, please contact sales@materialize.com.

Reviewer: In line with [this comment](https://materializeinc.slack.com/archives/C0538L6HFJM/p1682457957124089?thread_ts=1682456986.458229&cid=C0538L6HFJM), it'd be nice to provide two options here and soften the mention to sales: link to the community Slack + link to a "Talk with your team" form (_i.e._ soften the mention to sales).

Author: Made some changes based on a Slack discussion.

## How long does a free trial last?

14 days.

> If you want to continue using Materialize beyond 14 days, please contact sales@materialize.com. Otherwise, Materialize will delete your resources and data at the end of the trial period.

Reviewer: Same here. It seems unnecessary to mention this twice in a row, too! So maybe you can merge these two questions.

Author: I had them merged at one point, but the time limit felt buried in the mix of details around resource limits.

## How do I monitor credit usage?

> To see how many compute credits you are currently using, run the following query:

Reviewer 2: ```suggestion
To see your current rate of credit consumption, measured in credits per hour, run the following query:
```

Author: Done.

```sql
SELECT sum(s.credits_per_hour) AS credit_consumption_rate
FROM mz_cluster_replicas r

> JOIN mz_internal.mz_cluster_replica_sizes s ON r.size = s.size;To

Reviewer: ```suggestion
JOIN mz_internal.mz_cluster_replica_sizes s ON r.size = s.size;
```

```

> For example, if you start your free trial with the [getting started guide](/get-started/), you'll end up consuming `3.5` credits:

Reviewer 2: ```suggestion
For example, if you start your free trial with the [getting started guide](/get-started/), you'll end up consuming credits at a rate of `3.5` per hour:
```

Author: Thanks, @benesch. I think I'll also rename the alias for the `SELECT` output from `total_credits_per_hour` to `credit_consumption_rate`.

```nofmt
 total_credits_per_hour
------------------------
                    3.5
(1 row)
```

> ## Can I go over the credit limit?

Reviewer 2: ```suggestion
## Can I go over the credit rate limit?
```

Author: Done.

> No, you cannot go over the 4 credit limit at any time during your free trial. If you try to add a replica that puts you over the limit, Materialize will return an error.

Reviewer 2: ```suggestion
No, you cannot go over the rate limit of 4 credits per hour at any time during your free trial. If you try to add a replica that puts you over the limit, Materialize will return an error.
```

Author: Done.

If you need additional credits during your trial, please contact sales@materialize.com.

## How do I get help during my trial?

If you have questions about Materialize or need support, reach out to us in our [Community Slack](https://materialize.com/s/chat) or set up time to [chat with our team](http://materialize.com/convert-account/).

> ## How do I convert to a paid plan?

Reviewer: This also feels a little redundant.

Reviewer 3: Same - we mention connecting with sales multiple times. I think we can swap those `mailto:` links out for the convert account landing page and then remove a few mentions of contacting sales

Author: Hmm. I'm not exactly sure what changes to make here. Will think it through.

Author: Made some changes.
